pom.xml

```diff
@@ -141,6 +153,22 @@
       <version>0.25</version>
       <scope>test</scope>
     </dependency>
+    <!-- Managed by BOM -->
```

Comment:
Note that you _could_ also remove version overrides for some other plugins, such as `credentials`.

Reply:
yes, understood, although I'm considering it out-of-scope currently.